docs/Deploying-Pega-on-EKS.md

```diff
@@ -511,30 +514,13 @@ Configure the parameters so the pega.yaml Helm chart matches your deployment res
    | tier.name: ”stream” tier.service.domain: | Set the host name for the pega-stream service of the DNS zone.   | <ul><li>domain: "\<the host name for your stream service tier\>" </li><li>Your stream tier host name should comply with your networking standards. </li></ul>|
-   | pegasearch.image: | Specify the Pega-provided Docker `search` image that you downloaded and pushed to your Docker registry. | Image: "\<Registry host name:Port>/my-pega-search:\<Pega Platform version>"
+   | pegasearch: | For Pega Platform 8.6 and later, Pega recommends using the chart 'backingservices' to enable Pega SRS. To use this service, you must enable its use and provide the SRS URL for your Pega Infinity deployment. | <ul><li>externalSearchService: true</li><li>externalURL: pegasearch.externalURL For example, http://srs-service.mypega-eks-demo.svc.cluster.local </li></ul>
    | installer.image:        | Specify the Pega-provided Docker `installer` image that you downloaded and pushed to your Docker registry. | Image: "\<Registry host name:Port>/my-pega-installer:\<Pega Platform version>" |
    | installer. adminPassword:                | Specify an initial administrator@pega.com password for your installation.  This will need to be changed at first login. The adminPassword value cannot start with "@".     | adminPassword: "\<initial password\>"  |
 
-3. [For Pega Platform 8.6 and later] Applicable only when backingservices chart is configured.
-
-   For Pega Platform 8.6 and later installations in which you are configuring the backingservices Search and Reporting Service in your deployment, use a text editor to open the `pega.yaml` and update the following parameters in the chart based on your backing service configuration.
-
-   Chart parameter name   | Purpose   | Your setting
-   ---         | ---           | ---
-   `pegasearch.externalSearchService` | Set the `pegasearch.externalSearchService` as true to use Search and Reporting service as the search functionality provider to the Pega platform | true
-   `pegasearch.externalURL` | Set the `pegasearch.externalURL` value to the Search and Reporting Service endpoint url | `"http://<srs.deploymentName>.<namespace>.svc.cluster.local"` or `"http://<srs.deploymentName>.<namespace>"`
-
-   Example:
-
-   When backingservice is deployed into `mypega-EKS-demo` namespace and `pegasearch.externalSearchService` value is "srs-service", configure the `pegasearch` section in pega.yaml as below:
-
-   ```yaml
-   pegasearch:
-     externalSearchService: true
-     externalURL: "http://srs-service.mypega-EKS-demo.svc.cluster.local"
-   ```
+3. Save the file.
 
-4. Save the file.
+4. To use an internal Elasticsearch cluster (srs.srsStorage.requireInternetAccess:true) for your deployment, you must download the Makefile file from the repository ( path from root : pega-helm-charts/charts/backingservices/Makefile) and replace <YOUR_NAMESPACE> with the namespace you used for the deployment, then run `$ es-prerequisite`.
```

Comment:
is this correct?
then run `$ es-prerequisite`

I thought it was>
then run `$ make es-prerequisite`

Reply:
Yes it is `$ make es-prerequisite`. changed in the doc too.thanks for the catch.